eskip/string.go

```diff
@@ -14,12 +14,15 @@ type PrettyPrintInfo struct {
 }
 
 func escape(s string, chars string) string {
-
-	s = strings.Replace(s, "\\", "\\\\", -1)
+	s = strings.Replace(s, `\r`, "\r", -1)
```

Comment:
I believe, since we are escaping here, this should be the opposite:

```
s = strings.Replace(s, "\r", `\\r`, -1)
```

the line for the `\` is correct.

Reply:
Actually, this is a trick. I replace the literals with the actual newline char 10, then continue the escaping process, and then I convert the newline char back (10) to the `\` + `n` chars. That way, we don’t get double escapes.